data/repos.yml

```diff
@@ -258,6 +326,13 @@
   team: "#tech-interaction-and-personalisation"
   production_hosted_on: aws
 
+# Looks useful and worth keeping?
+- repo_name: fake-ckan_harvest_source
```

Comment:
@kentsanggds

Reply:
Useful for testing using a mock harvest source